common/script/fns/cron.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(user, options) {
   }
   user.auth.timestamps.loggedin = new Date();
   user.lastCron = now;
+  if (_.isFinite(+user.preferences.timezoneOffset)) {
```

Comment:
It returns true it it's a number (positive, negative, or zero). Returning true on zero is important so that GMT timezones aren't ignored.  https://lodash.com/docs#isFinite